sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/Platform.java

```diff
@@ -336,6 +336,22 @@ public IOS_AARCH64() {
         }
     }
 
+    /**
+     * Supported leaf platform: iOS on x86 64-bit.
+     *
+     * @since 21.1
+     */
+    final class IOS_AMD64 implements IOS, AMD64 {
```

Comment:
Can we somehow explain to the user, via a comment here, and potentially in the list of platforms that this platform is experimental and that it supports only subsets of Java?

Reply:
This needs to be on "interface IOS" level, since this is true for all iOS targets.

Reply:
Yeah currently all iOS targets (well, there is only one iOS target in master though) cannot produce an executable.